alibi/explainers/anchors/anchor_base.py

```diff
@@ -351,7 +351,9 @@ def draw_samples(self, anchors: list, batch_size: int) -> Tuple[tuple, tuple]:
             if anchor not in self.state['t_order']:
                 self.state['t_order'][anchor] = list(anchor)
 
-        sample_stats, pos, total = [], (), ()  # type: List, Tuple, Tuple
+        sample_stats: List = []
+        pos: Tuple = tuple()
```

Comment:
Is the `tuple()` preferred to `()`?

Reply:
Just noticed we tend to mostly do `tuple()` elsewhere...

Reply:
I tend to use tuple because `()` is ambiguous and can create errors in certain contexts.